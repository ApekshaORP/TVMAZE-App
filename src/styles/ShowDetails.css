.container {
  display: flex;
  align-self: center;
  position: relative;
  flex-direction: row;
  flex-wrap: wrap;
  margin: 0 auto;
  border-bottom: 1px solid rgba(151,151,151,0.21);
  }
  
  .container img {
  width: 300px;
height: 50%;}
  
  .slider_container{
    padding: 2rem;
    background-color: #070d18 ;

}
  * {
  margin: 0;
  padding: 0;
}
.container {
  display: flex;
  justify-content: center;
  font-weight: bold;
  vertical-align: top;
}
.main {
  flex: 1;
  text-align:start;
  padding: 2rem;
    color: white;
  
}
.image {
  height: 50%;
  padding: 2rem;
justify-content: center;
}
.image img{
  border:2px solid #192641;
border-radius: 50px;
}
.details_data{
  font-size: 1rem;
    font-weight: normal;
    margin-top: 20px;
    margin-left: 10px;
}
.details_rating{
  margin-top: 6px;
  margin-left: 10px;
  font-weight: normal;
}
.list_item{
  display: flex;
  align-items: center;
  flex-wrap: wrap;
}