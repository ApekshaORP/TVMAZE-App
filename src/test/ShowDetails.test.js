import React from 'react';
import ShowDetails from '../component/ShowsDetails';
import {shallow} from 'enzyme';

const SetUp=(props={})=>{
    const component=shallow(<ShowDetails {...props}/>)
    return component
}
describe('Show Details component', () => {
   it ('It should match snapshot',()=>{
    const wrapper=SetUp()
    // const wrapper =component.find('.container');
    expect(wrapper).toMatchSnapshot();   
})
  });
