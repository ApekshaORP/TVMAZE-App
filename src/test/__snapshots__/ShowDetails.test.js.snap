// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Show Details component It should match snapshot 1`] = `ShallowWrapper {}`;
